README.md

```diff
@@ -73,7 +73,8 @@ Every time a celery task is executed, the tenant object of the `connection` obje
 For some use cases, this can introduce significant performance hit.
 
 In such scenarios, you can pass `tenant_cache_seconds` argument to the `@app.task()` decorator. This will
-cause the tenant objects to be cached for given period of time. `0` turns this off.
+cause the tenant objects to be cached for given period of time. `0` turns this off. You can also enable cache globally
+by setting `CELERY_TASK_TENANT_CACHE_SECONDS`.
```

Comment:
The `CELERY_` prefix of the setting is actually user-defined. If you configure your app to load config with `CELERY` namespace, then this will be true. But doesn't have to be.